Project/Project/Areas/Identity/Pages/Account/Login.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@page
+@model Microsoft.AspNetCore.Identity.UI.V4.Pages.Account.Internal.LoginModel
+
+@{
+Layout = "/Areas/Identity/Pages/_AuthLayout.cshtml";
+}
+
+<div class="row">
+    <div class="col-md-12">
+        <section>
+            <form id="account" method="post">
+                <h2 class="text-center fw-bold mb-5 text-white">Welcome Back!</h2>
+
+                <div asp-validation-summary="ModelOnly" class="text-danger" role="alert"></div>
+                <div class="form-floating mb-3">
+                    <input asp-for="Input.Email" class="form-control rounded-pill" autocomplete="username" aria-required="true" placeholder="Username" />
+                    <label asp-for="Input.Email" class="form-label text-black">Email</label>
+                    <span asp-validation-for="Input.Email" class="text-danger"></span>
+                </div>
+                <div class="form-floating mb-3">
+                    <input asp-for="Input.Password" class="form-control rounded-pill" autocomplete="current-password" aria-required="true" placeholder="Password" />
+                    <label asp-for="Input.Password" class="form-label text-black">Password</label>
+                    <span asp-validation-for="Input.Password" class="text-danger"></span>
+                </div>

```

Comment:
The inputs look diffrent in the conecpt, also the spacing is not right

<img width="585" alt="image" src="https://github.com/Maxiboy441/PixelPrime/assets/116082321/e2a68662-6983-4a33-b3f0-27c78bd0479c">

Its not spaced out correctly